tests/integration_tests/scheduler/test_openpbs_driver.py

```diff
@@ -117,11 +116,8 @@ def test_openpbs_driver_with_poly_example(tmp_path, source_root, monkeypatch):
         parser,
         [
             ENSEMBLE_EXPERIMENT_MODE,
-            "--enable-scheduler",
```

Comment:
Will this use scheduler without the flag, and without the ```use_scheduler```-mark? Or is it set as the default for the TORQUE queue system?